database/resources/project-ideas.json

```diff
@@ -46,5 +46,12 @@
     "url": "https://www.interviewbit.com/blog/artificial-intelligence-projects/",
     "category": "resources",
     "subcategory": "project-ideas"
+  },
+  {
+    "name": "Cloud Computing Projects",
+    "description": "This website provides free Cloud Computing projects for beginners to advanced with source code.",
```

Comment:
```suggestion
    "description": "This website provides free Cloud Computing projects for beginners to advanced level with source code.",
```

Reply:
@rupali-codes yes , I've made the necessary changes.
Kindly review it.

Reply:
thanks